src/platform/platform.ts

```diff
@@ -1180,7 +1181,7 @@ export function setupPlatform(doc: HTMLDocument, platformConfigs: {[key: string]
   // set values from "document"
   const docElement = doc.documentElement;
   plt.setDocument(doc);
-  plt.setDir(docElement.dir, false);
+  plt.setDir(docElement.dir === 'rtl' ? 'rtl' : 'ltr', false);
```

Comment:
Let's not dynamically set this since it's got a good potential to force relayout. You can use `setDir`, but it'd better to have the index file already hardcoded with the value, and all of our starters should already have dir set (if not let's fix that).